### PR TITLE
Add 06-contributing

### PR DESCRIPTION
**Pull request by BobbyRBruce**

### CODE REVIEW

**Comment by BobbyRBruce** [comment]
@powerjg : This is ready for review.

File: slides/06-Contributing/01-contributing-img/our-strategy.svg.drawio
Comment (powerjg) [inline slides/06-Contributing/01-contributing-img/our-strategy.svg.drawio:1]: Why is this file here?

I think you meant to call the other one `our-strategy.drawio.svg`

File: slides/06-Contributing/02-testing.md [hunk 1]
```diff
@@ -2,14 +2,292 @@
...
+        "gem5",
+        "m5_util",
+        "configs",
+        "simple_binary_run.py",
+    ),
+    config_args=[ # The arguments to pass to the config file.
+        "x86-m5-exit",
+        "--resource-directory",
+        resource_path,
+    ],
```
Comment (powerjg) [inline slides/06-Contributing/02-testing.md:191]: Missing `)`, I think. Something is wrong with the syntax here.

File: slides/06-Contributing/01-contributing-img/our-strategy.svg.drawio
Comment (BobbyRBruce) [reply slides/06-Contributing/01-contributing-img/our-strategy.svg.drawio:1]: File removed.

File: slides/06-Contributing/02-testing.md (hunk 1 above)
Comment (BobbyRBruce) [reply slides/06-Contributing/02-testing.md]: Fixed

**Review by powerjg — APPROVED** [review]
